fera_starter.py

```
@@ -1,3 +1,27 @@
+        exe_path = os.path.join(os.getcwd(), 'FERA', 'FERA_LINUX', 'fera')
+    
+    pathdb = None
+    for filename in os.listdir(os.path.join(os.getcwd(), 'FERA')):
+        if(os.path.isdir(filename)):
+            continue
+        filenamenoext, extension = os.path.splitext(filename)
+        if ("fera" in filenamenoext.lower() and ".db" == extension.lower()):
+            pathdb = os.path.join(os.getcwd(), 'FERA', filename)
+            break
+    
+    if(pathdb!=None):
```

Ruff

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)